README.md

```diff
@@ -488,7 +488,7 @@ When using Attachments, ensure content is provided as a byte array. It's importa
 If you wish to develop the Local Terminal API integration parallel to your encryption implementation, you can opt for the unencrypted version. Be sure to remove any encryption details from the CA terminal config page.
```

Comment:
We could add a note here: `Consider this ONLY for development and testing on locahost`